modules/widgets/google-translate.php

```diff
@@ -104,7 +119,10 @@ public function form( $instance ) {
 	 */
 	public function update( $new_instance, $old_instance ) {
 		$instance = array();
-		$instance['title'] = ( ! empty( $new_instance['title'] ) ) ? sanitize_text_field( $new_instance['title'] ) : '';
+		$instance['title'] = wp_kses( $new_instance['title'], array() );
```

Comment:
That's the way it's done [in other widgets](https://github.com/Automattic/jetpack/blob/master/modules/widgets/my-community.php#L133), but I'm happy to change it if needed!